packages/core/src/components/Tipseen/Tipseen.tsx

```diff
@@ -152,6 +152,7 @@ const Tipseen: VibeComponent<TipseenProps> & {
                 [styles.dark]: closeButtonTheme === "dark" || closeButtonTheme === "fixed-dark"
               })}
               onClick={onClose}
+              data-testid={getTestId(ComponentDefaultTestId.TIPSEEN_CLOSE_BUTTON)}
```

Comment:
We usually add data-testid for the root elements of components and try to avoid having many data-testids on a lot of elements. We encourage using aria/role attributes as it's the common best practice, in this example you can use the aria-label: `[data-testid="tipseen"] [aria-label="Close"]`